parenthetics.py

```diff
@@ -0,0 +1,13 @@
+
+
+class Parenthentic(object):
+
+    def isequal(self, text):
+        count = 0
+        for par in text:
+            if par == '(':
+                count += 1
+            if par == ')':
+                count -= 1
+                return count
+        return count
```

Comment:
Two questions:

1.  Why a class?  Do you use `self` for anything at all?  If not, why make this a class with a method, instead of just a simple function?

2.  Does this actually work?  Your tests pass, but try this test:

```python
def test_better():
    text = Parenthentic()
    assert text.isequal("(()())") == 0
```

It will fail.